WalletWasabi/WabiSabi/Client/AliceClient.cs

```diff
@@ -213,23 +213,12 @@ public async Task TryToUnregisterAlicesAsync(CancellationToken cancellationToken
 		{
 			Logger.LogTrace(e);
 		}
+		catch (Exception e) when (e is HttpRequestException or WabiSabiProtocolException)
+		{
+			Logger.LogDebug($"Unregistration failed for coin '{SmartCoin.Coin.Outpoint}'.", e);
+		}
 		catch (Exception e)
 		{
-			if (e is HttpRequestException && e.InnerException is WabiSabiProtocolException wpe)
-			{
-				switch (wpe.ErrorCode)
-				{
-					case WabiSabiProtocolErrorCode.RoundNotFound:
-						SmartCoin.CoinJoinInProgress = false;
```

Comment:
Same question.

Now we don't assign 
`SmartCoin.CoinJoinInProgress = false;` in case of `ProtocolException`s

Reply:
If the unregistration was not successful we should not indicate that the coin is not in the CJ. I believe it was wrong conceptually. 

In case of `RoundNotFound` we will quickly find yourself here: 

https://github.com/molnard/WalletWasabi/blob/fixwarn/WalletWasabi/WabiSabi/Client/CoinJoinClient.cs#L333

Setting the `CoinJoinInProgress` to false.

Reply:
Make sense.